packages/examples/packages/wasm/.eslintrc.js

```diff
@@ -1,28 +1,17 @@
-const { resolve } = require('path');
-
 module.exports = {
   extends: ['../../.eslintrc.js'],
 
   parserOptions: {
     tsconfigRootDir: __dirname,
   },
 
-  overrides: [
```

Comment:
?

Reply:
So should we remove the tsconfig entirely from the WASM code?

Reply:
I'm not sure but I can if you prefer?